client/src/components/Home.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { Button } from 'antd';
```

Comment:
I think I wanna get away from antd. We could either fix it up after merging this, or we could just hop straight to tailwind in this PR :shrug: